python/tvm/autotvm/graph_tuner/graph_executor/base_graph_executor.py

```diff
@@ -0,0 +1,410 @@
+# pylint: disable=too-many-arguments,too-many-locals,too-many-statements,too-many-instance-attributes,too-many-branches,too-many-nested-blocks
```

Comment:
Please don't put these on top of a file. Move them under specific classes or functions, where pylint should be disabled for a certain reason.